decentraland/unity-explorer
# Fix: disabled street view section

## What does this PR change?

Fix #851 disables street view mode in navmap as suggested in the issue
Fix #961 
...

## How to test the changes?



1. Launch the explorer
2. Open the navmap
3. Verify that the street view section is not present anymore

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
